ansible_builder/requirements.py

```diff
@@ -39,7 +39,7 @@ def sanitize_requirements(collection_py_reqs):
         try:
             for req in requirements.parse('\n'.join(lines)):
                 if req.specifier:
-                    req.name = safe_name(req.name)
+                    req.name = safe_name(req.name).lower()
```

Comment:
Fix LGTM, though while we're "in the neighborhood", we should probably either switch to `Prepared.normalize()` from `importlib.resources` (which handles the character replacements as well as the case internally) or just embed its impl directly, since `Prepared` doesn't seem to be documented :frowning_face:. `pkg_resources` is deprecated, and part of `setuptools`, which IIRC will not be installed by default anymore in new virtualenvs and other places, so we can't necessarily rely on its presence (nor should we declare a runtime dep on it).

Reply:
I agree, let us switch to `importlib.metadata` just like https://github.com/ansible/ansible-runner/pull/1224